### DISCUSSION TITLE
How to load model on Android

### DISCUSSION THREAD

**Discussion by eldhosekurian** [asus4/tf-lite-unity-sample#354] [Q&A]
I made a custom tflite model for image detection with the help of this repository. it works perfectly on my unity but when i export it as apk and run it on my android device, there is no prediction.It is not real time. but I captured image in one scene and display the predicted class in another scene. Is there any addtional scripts that i need to use ? please do give a clarity because i have been trying this for one week straight and nobody responded. is this related to build.py ?

**Comment by asus4** [comment]
@eldhosekurianmathewofficial 

It is not related to TensorFlow, but I noticed an issue here. 

https://github.com/eldhosekurianmathewofficial/objectdetection/blob/f28225b22c3361d1ceea3f1c9ddf0d8ef07e973b/Assets/Samples/Detection/Scripts/PythonBridge.cs#L58

File.ReadAllBytes from streaming assets doesn't work on Android. Use another method, such as UnityWebRequest, to get file bytes.
